src/themes/advent-of-code.css

```diff
@@ -0,0 +1,83 @@
+:root {
+	--background: #0f0f23;
+	--text: #cccccc;
+	--link: #009900;
+	--accent: #ffff66;
+}
+
+/* for some reason importing this at the top dims everything, curious */
+@import url("https://fonts.googleapis.com/css2?family=Source+Code+Pro:wght@200&display=swap");
+
+html,
+body {
+	overflow: hidden;
+	height: 100vh;
+	margin: 0;
+	font-family: "Source Code Pro", monospace;
+	color: var(--text);
+	background-color: var(--background);
+}
+
+/* the entire list of messages */
+[data-twitch-chat] {
+	list-style: none;
+	margin: 0;
+	padding: 0;
+	height: 100vh;
+	overflow: auto;
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-end;
+}
+
+[data-twitch-chat] > * {
+	margin-top: 1rem;
+}
+
+[data-twitch-chat] > *:last-child {
+	margin-bottom: 2rem;
+}
+
+/* single message block */
+[data-twitch-message] a {
+	text-decoration: none;
+	color: var(--link);
+}
+
+/* replying to line on top of a message */
+[data-twitch-message-reply] {
+	opacity: 0.6;
+}
```

Comment:
Thank you! Suggestions are very welcome as this is pretty bare-bones and I haven't thought of many things yet!

Reply:
Hmmm. apparently that doesn't work on a `<small>` tag (the reply element is one).
I tried changing it to a `<p>` just to see what happens then, and it works. (Chrome 97)
I'll leave it alone for this PR, and it can be addressed properly later.

Reply:
That should be because `small` is an inline element by default - try setting `display: block;` and the truncation should work 👍🏻

Reply:
yup! You're bang on, already pushed 👍 
I figured it out by reading some MDN docs, glad they have an "applies to" section.